skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -395,7 +429,6 @@ def repo_path_for_inference():
 
 
 @pytest.mark.network
-@pytest.mark.inference
```

Comment:
this is probably removed by mistake

Reply:
Thank you! I removed that somehow by mistake. I added it back.